### ISSUE TITLE
Creating a new window with Alt+F2 while in fullscreen creates a large terminal

### ISSUE THREAD

**Issue by ElvenSpellmaker** [mintty/mintty#633]
1.) Have a terminal in fullscreen (Alt+Enter)
2.) Press Alt+F2 to create a new window
3.) The terminal is the size of the fullscreen terminal (but isn't fullscreen itself)

I would expect it to be the size of the terminal _before_ being made fullscreen (which is known as Alt+Enter takes it back to the size it was before), to mimic the behaviour of when the terminal isn't in fullscreen.

**Comment by mintty** [comment]
> to be the size of the terminal before being made fullscreen
which it actually does for me.
Please provide your mintty version, Windows version, and mintty config file.

**Comment by ElvenSpellmaker** [comment]
I'll run a Cygwin update now and see if that fixes it.

**Comment by ElvenSpellmaker** [comment]
Mintty 2.7.4
Windows 10 Pro x64

```
BoldAsFont=yes
CursorType=block
Transparency=medium
BackgroundColour=0,0,64
OpaqueWhenFocused=yes
Font=Consolas
FontHeight=16
Locale=en_GB
Charset=UTF-8
Columns=92
Rows=30
Scrollbar=right
ScrollMod=off
Term=xterm-256color
BellSound=yes
AllowBlinking=yes
FontSmoothing=full
CtrlShiftShortcuts=yes
BellFlash=no
ClicksPlaceCursor=yes
PgUpDnScroll=yes
ClicksTargetApp=no
Blue=17,17,191
ForegroundColour=191,191,191
BoldAsColour=no
```

**Comment by mintty** [comment]
Intending to close as it's not reproducible.

**Comment by mintty** [comment]
Reproduced on Windows 7.

**Comment by mintty** [comment]
Please check.
Note that Shift+Alt+F2 would not clone the size. Also fixed: alternating use of Alt+F2 and Shift+Alt+F2.

**Comment by mintty** [comment]
Released 2.7.6.

**Comment by mintty** [comment]
Released 3.1.7.

**Comment by ElvenSpellmaker** [comment]
Is there anyway to clone without inheriting the full screen?

Pressing shift as well as Fn+Alt+F2 doesn't seem to work for me.

**Comment by mintty** [comment]
Shift+Alt+F2 does not seem to work as described, to be checked.
Also, a definable key function could be added for non-cloning new window.

**Comment by mintty** [comment]
Released 3.3.0.
